pos_restricted_customer_list/static/src/js/pos_restricted_customer_list.js

```diff
@@ -0,0 +1,57 @@
+odoo.define("pos_restricted_customer_list.point_of_sale.models", function(require) {
```

Comment:
```suggestion
odoo.define("pos_restricted_customer_list.models", function(require) {
```

Also this file should be renamed to `models.js`, for convenience.

it's also good to return the PosModels object at the end. I know it's unlikely some other module will extend it, but it's nice ;)